### PR TITLE
Readme Update

### PR DESCRIPTION
**Pull request by shvamath**
Readme update for packages which I had to install

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -252,6 +251,11 @@ safe. SeaHorn returns `sat` when `__VERIFIER_error()` is reachable and
 the program is unsafe. `sassert()` method is defined in
 `seahorn/seahorn.h`.
 
+## Ubutnu 18.04 Packages ##
+Before proceeding with installation make sure you have all of the following packages:
```
Comment (kuhar) [inline README.md]: I'd say these are suggested packages you may need

File: README.md [hunk 2]
```diff
@@ -252,6 +251,11 @@ safe. SeaHorn returns `sat` when `__VERIFIER_error()` is reachable and
...
+* `sudo apt install cmake build-essential ninja-build llvm-6.0 clang-6.0 lld-6.0 git-svn libboost-dev subversion g++-7-multilib gcc-multilib lib32stdc++7 libgmp-dev libgmpxx4ldbl libgraphviz-dev`
```
Comment (kuhar) [inline README.md]: git-svn is not strictly necessary, while git obviously is

File: README.md [hunk 3]
```diff
@@ -252,6 +251,11 @@ safe. SeaHorn returns `sat` when `__VERIFIER_error()` is reachable and
...
+
+You may also need to install the following packages if getting warning: `sudo apt install libncurses5-dev ncurses-doc`
```
Comment (kuhar) [inline README.md]: why not merge it with the previous list?

Comment (caballa) [reply README.md]: Why not clang-5.0 rather than 6.0. Also, why do we need llvm? we install our own from sources.
ninja is optional.

Comment (kuhar) [reply README.md]: the lld linker started working around the ~6.0 release; and it makes sense to take the newest package your distro offers

Comment (kuhar) [reply README.md]: I'd also argue that the default ld.bfd linker is horribly slow and nobody should ever use it in 2019

Comment (caballa) [reply README.md]: but it can happen (maybe not with 6.0) that clang generates some bitcode that our 5.0 frontend could not digest

Comment (kuhar) [reply README.md]: the intention is this is for building seahorn itself, not for generating bitcode

File: README.md (hunk 1 above)
Comment (kuhar) [reply README.md]: or something like: suggested packages for building seahorn

**Comment by agurfinkel** [comment]
Thank you! I've merged it manually. Closing the PR here.

**Pull request closed without merging**